management_api_app/service_bus/service_bus.py

```diff
@@ -0,0 +1,27 @@
+from azure.identity.aio import DefaultAzureCredential
+from azure.servicebus import ServiceBusMessage
+from azure.servicebus.aio import ServiceBusClient
+from core import config
+
+class ServiceBus():
+    """
+    Implements methods to operate the service bus in the core infrastructure of Azure TRE.
+    """
+
+    async def send_resource_request_message(self, resource_request_message: str):
+        """
+        Sends the given message to the resource request queue.
+
+        :param resource_request_message: The message to send.
+        :type resource_request_message: str
+        """
+        credential = DefaultAzureCredential()
```

Comment:
should credentials be read in class initialize?

Reply:
Credential instance is closed in the end of the method call so that would render the ServiceBus instance nonfunctional after one-time use.